lib/Parser/Parse.cpp

```diff
@@ -3262,10 +3265,12 @@ LFunction :
             m_pscan->Scan();
             if (m_token.tk == tkLParen)
             {
-                return ParseImportCall<buildAST>();
```

Comment:
I think this would read a little cleaner using `ChkCurTok(tkLParen, ERRnoLparen);`